org.gnome.gitlab.ilhooq.Bookup.json

```diff
@@ -60,7 +60,7 @@
                     "type" : "git",
                     "url" : "https://gitlab.gnome.org/ilhooq/bookup.git",
                     "tag" : "v0.1.0",
-                    "commit": "62e17a0ca070e613cabc88a4b09f17ed1d198636"
+                    "commit": "4475d66c643eb5654f5da392f842ce1016224d24"
```

Comment:
retagging is a bad practice.

Reply:
I have to change the tag or it's ok ?

Reply:
in general you never retag a published tag. It's bad practice. this is not flathub specific.